mods/action.js

```diff
@@ -181,6 +181,13 @@ function buildEditResponse(res) {
 }
 
 ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
+    if (/\.test\.local$/.test(req.params.domain)) {
```

Comment:
@gwicke Switching all test to labs is a amount of work, because we need to create same pages as we use for testing, alter all revision numbers, and sometimes pages need to have a pretty specific history. Additionally, to avoid hacks like this there is another PR: https://github.com/wikimedia/restbase/pull/324